config/canonical.json

```diff
@@ -31055,10 +31065,15 @@
   },
   "shop/supermarket|Пятёрочка": {
     "count": 4002,
+    "countryCodes": ["ru"],
```

Comment:
According to the wikipedia article (and the rough translation of it) this brand also operates in Ukraine and Kazakhstan. [Overpass](https://overpass-turbo.eu/s/D1S)

Reply:
But in Wikipedia article said that network close their stores in Ukraine in 2007 and made a rebrand at Kazakhstan in the same year

Reply:
> 
> 
> According to the wikipedia article (and the rough translation of it) this brand also operates in Ukraine and Kazakhstan. [Overpass](https://overpass-turbo.eu/s/D1S)
But in Wikipedia article said that network close their stores in Ukraine in 2007 and made a rebrand at Kazakhstan in the same year

Reply:
This is interesting. I will extract this case into an issue and merge the PR